### PR TITLE
Should be DiscoverProtocol not ExchangeProtocol

### PR DESCRIPTION
**Pull request by ma-sonr**
<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: pkg/motor/x/discover/protocol.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func New(ctx context.Context, host host.SonrHost, cb common.MotorCallback, optio
 		opt(opts)
 	}
 	opts.Apply(protocol)
-	log.Println("✅  ExchangeProtocol is Activated")
+	log.Println("✅  DiscoveryProtocol is Activated")
```
Comment (ntindle) [inline pkg/motor/x/discover/protocol.go]: Can you change it to discover?

**Comment by ntindle** [comment]
@ma-sonr good spot

File: pkg/motor/x/discover/protocol.go (hunk 1 above)
Comment (ma-sonr) [reply pkg/motor/x/discover/protocol.go]: Yeah

**Review by ntindle — APPROVED** [review]

**Review by joshLong145 — APPROVED** [review]

**Review by mcjcloud — APPROVED** [review]
lgtm

**Pull request merged**